webpack.config.dev.js

```diff
@@ -2,13 +2,10 @@ const webpack = require('webpack');
 const deepAssign = require('deep-assign');
 const baseConfig = require('./webpack.config');
 
-const devConfig = deepAssign({}, baseConfig);
-
-// source mapping
-devConfig.devtool = 'cheap-module-source-map';
-
-// apply plugins
-devConfig.plugins = baseConfig.plugins || [];
-devConfig.plugins.push(new webpack.LoaderOptionsPlugin({ debug: true }));
+const devConfig = deepAssign({}, baseConfig, {
```

Comment:
There's another config called `webpack-merge` that we can use going forward instead of `deep-assign`. Just a thought. Can be a new diff. 😁

Reply:
a01743c